micropub.php

```diff
@@ -173,23 +179,26 @@ private static function authorize() {
 
 		// look for a user with the same url as the token's `me` value. search both
 		// with and without trailing slash.
-		foreach ( array_merge( get_users( array( 'search' => $me ) ),
-							   get_users( array( 'search' => $me . '/' ) ) )
-				  as $user ) {
-			if ( untrailingslashit( $user->user_url ) == $me ) {
+		foreach ( array_merge(
+			get_users( array( 'search' => $me ) ),
+			get_users( array( 'search' => $me . '/' ) )
+		)
```

Comment:
oh man this kind of indentation - an inner paren all the way back at the top level - is so problematic and misleading when reading code! any way we can loosen PHPCS a bit with indentation rules like this?

Reply:
friendly ping

Reply:
ping